### ISSUE TITLE
FormSelectable Interface

### ISSUE THREAD

**Issue by apaganobeleno** [gobuffalo/tags#19]
I was thinking that we could provide an interface that allows to use any struct as an option on the SelectTag, form the top of my head something like:

```go
type FormSelectable interface {
    SelectValue() string
    SelectLabel() string
}
``` 

Would allow any of our models to be used as a select option, and we could still support maps and SelectOption to provide flexibility.

@markbates thoughts on this one ?

**Comment by markbates** [comment]
Sounds like a plan to me.

**Comment by apaganobeleno** [comment]
Closed with #21
